web/src/containers/admin/FederalAdmin.js

```diff
@@ -33,56 +32,66 @@ const FederalAdmin = ({
   const [selectedAffiliation, setSelectedAffiliation] = useState();
 
   const [manageModalDisplay, setManageModalDisplay] = useState(false);
-  const [confirmationModalDisplay, setConfirmationModalDisplay] = useState(
-    false
-  );
+  const [confirmationModalDisplay, setConfirmationModalDisplay] =
+    useState(false);
 
   const [limitedRoleTypes, setLimitedRoleTypes] = useState(roleTypes);
 
   useEffect(() => {
-    async function fetchAffiliations() {
-      await affiliations(currentState.id, activeTab);
-    }
-    
-    if(activeTab !== 'letters') {
-      setIsFetching(true);
-      fetchAffiliations().then(() => setIsFetching(false));      
-    }
+    (async () => {
```

Comment:
nice. good cleanup

Reply:
honestly it was kind of a hail mary because I had seen a post that that was a better way to handle it, but I do like the look of it better